### PR TITLE
search 

### PR DESCRIPTION
**Pull request by nietongyu**
Added a skeleton search function in message.

### CODE REVIEW

**Comment by jiangzhengwenjz** [comment]
Please merge master into your branch and resolve merge conflicts. 
```shell
git fetch origin
git merge origin/master
```

**Comment by nietongyu** [comment]
Merged

File: app/src/main/java/com/example/upeers/MainActivity.kt [hunk 1]
```diff
@@ -4,6 +4,13 @@ import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.View
+<<<<<<< HEAD
+import com.example.upeers.message.Message
+import com.example.upeers.msglist.MessageListActivity
+
+const val EXTRA_MSG = "com.example.message.Message"
+=======
+>>>>>>> origin/master
```
Comment (jiangzhengwenjz) [inline app/src/main/java/com/example/upeers/MainActivity.kt]: It's not resolved

File: app/src/main/java/com/example/upeers/MainActivity.kt [hunk 2]
```diff
@@ -17,4 +24,16 @@ class MainActivity : AppCompatActivity() {
...
+    fun goToMessage(view: View) {
+        val intent = Intent(this, Message::class.java).apply {
+
+        }
+        startActivity(intent)
+    }
+
+
+=======
+>>>>>>> origin/master
```
Comment (jiangzhengwenjz) [inline app/src/main/java/com/example/upeers/MainActivity.kt]: It's not resolved

File: app/src/main/java/com/example/upeers/MainNaviActivity.kt [hunk 3]
```diff
@@ -63,6 +65,14 @@ class MainNaviActivity : AppCompatActivity() {
...
 
-    //     return super.onCreateOptionsMenu(menu)
-    // }
+//        return super.onCreateOptionsMenu(menu)
+//    }
+
+    fun onClickMsgUserCard(view: View) {
+        val intent = Intent(this, MessageListActivity::class.java).apply {
+            // should use putExtra here I guess
+//                val remoteUserName = it.username.text
```
Comment (jiangzhengwenjz) [inline app/src/main/java/com/example/upeers/MainNaviActivity.kt]: please start comment from the current indent level (please check other files)

File: .idea/codeStyles/Project.xml [hunk 4]
```diff
@@ -1,125 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
```
Comment (jiangzhengwenjz) [inline .idea/codeStyles/Project.xml:1]: If you think this file should not be tracked, please edit `.gitignore`
(But is there a reason to untrack this file? )

File: .idea/navEditor.xml [hunk 5]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jiangzhengwenjz) [inline .idea/navEditor.xml]: Do you think this file should be tracked?

**Review by jiangzhengwenjz — CHANGES_REQUESTED** [review]
Please resolve the comments

Another thing I forgot to mention last time: please don't use chinese comments in any files, thanks

File: app/src/main/res/layout/layout_message_bubble.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:background="@drawable/rounded_rectangle_orange"
+        android:maxWidth="@dimen/max_msg_width"
+        android:padding="8dp"
+        android:text="hi man, how are you?"
+        android:textColor="#ffffff" />
+
+</LinearLayout>
```
Comment (jiangzhengwenjz) [inline app/src/main/res/layout/layout_message_bubble.xml]: newline

File: app/src/main/java/com/example/upeers/msglist/MessagesAndUser.kt [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+class Message(val message: String,
+              val sender: User,
+              val createdAt: Long) {
+
+}
+
+class User (val nickname: String, val porfileUrl: String) {
+
+}
```
Comment (jiangzhengwenjz) [inline app/src/main/java/com/example/upeers/msglist/MessagesAndUser.kt]: newline

File: app/src/main/java/com/example/upeers/msglist/MessageListAdapter.kt [hunk 8]
```diff
@@ -0,0 +1,7 @@
+package com.example.upeers.msglist
+
+import android.content.Context
+
+class MessageListAdapter(private val mContext:Context,
+                         private val mMessageList: List<Message>) {
+}
```
Comment (jiangzhengwenjz) [inline app/src/main/java/com/example/upeers/msglist/MessageListAdapter.kt]: newline

File: app/src/main/java/com/example/upeers/ui/message/MessageUserInfoAdapter.kt [hunk 9]
```diff
@@ -25,6 +30,9 @@ open class MessageUserInfoAdapter(private val myDataSet: List<MessageUserInfo>)
         val cardView = LayoutInflater.from(parent.context)
             .inflate(R.layout.layout_message_contact_card, parent, false) as View
 
+//        cardView.setOnClickListener {
+//        }
```
Comment (jiangzhengwenjz) [inline app/src/main/java/com/example/upeers/ui/message/MessageUserInfoAdapter.kt]: comment should start from the current indent level

**Review by jiangzhengwenjz — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/upeers/ui/message/MessageUserInfoAdapter.kt (hunk 9 above)
Comment (nietongyu) [reply app/src/main/java/com/example/upeers/ui/message/MessageUserInfoAdapter.kt]: do you have any idea how to fix this on android studio?

**Review by jiangzhengwenjz — APPROVED** [review]

**Pull request merged**